cluster/addons/kube-proxy/kube-proxy-ds.yaml

```diff
@@ -44,6 +44,21 @@ spec:
         - /bin/sh
         - -c
         - kube-proxy {{cluster_cidr}} --resource-container="" --oom-score-adj=-998 {{params}} 1>>/var/log/kube-proxy.log 2>&1
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: 10256
+            scheme: HTTP
+          initialDelaySeconds: 15
+          timeoutSeconds: 15
+          successThreshold: 1
+          failureThreshold: 2
+        readinessProbe:
+          httpGet:
+            path: /healthz
+            port: 10256
+            scheme: HTTP
+          timeoutSeconds: 1
```

Comment:
please use:
```
          initialDelaySeconds: 15
          timeoutSeconds: 15
```
for the RP too.

Reply:
@neolit123 oops sorry!

Reply:
still missing:
`initialDelaySeconds: 15` for RP.